<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Expenses Visualizer</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="main_tab_widget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="settings_tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1081</width>
          <height>681</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QTabWidget" name="sub_settings">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="sett_import">
            <attribute name="title">
             <string>Import</string>
            </attribute>
            <widget class="QWidget" name="formLayoutWidget">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>1041</width>
               <height>101</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QPushButton" name="but_import_dir">
                <property name="text">
                 <string>Import directory</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="import_dir">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>File type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
                <property name="rightMargin">
                 <number>700</number>
                </property>
                <item>
                 <widget class="QComboBox" name="cb_file_type">
                  <property name="currentText">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Date format</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="rightMargin">
                 <number>500</number>
                </property>
                <item>
                 <widget class="QComboBox" name="cb_date_format"/>
                </item>
                <item>
                 <widget class="QLabel" name="date_format">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="but_import">
             <property name="geometry">
              <rect>
               <x>920</x>
               <y>590</y>
               <width>111</width>
               <height>29</height>
              </rect>
             </property>
             <property name="text">
              <string>Import</string>
             </property>
            </widget>
            <widget class="QTableWidget" name="data_preview_tab">
             <property name="geometry">
              <rect>
               <x>240</x>
               <y>130</y>
               <width>811</width>
               <height>261</height>
              </rect>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
            </widget>
            <widget class="QWidget" name="formLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>130</y>
               <width>221</width>
               <height>301</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Date</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="date_col"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Description</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="description_col"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Amount</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="amount_col"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Balance</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="balance_col"/>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="sett_categories">
            <attribute name="title">
             <string>Manage categories</string>
            </attribute>
            <widget class="QTableWidget" name="sett_category_aliases_tab">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>351</width>
               <height>441</height>
              </rect>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QTableWidget" name="sett_categories_tab">
             <property name="geometry">
              <rect>
               <x>370</x>
               <y>10</y>
               <width>691</width>
               <height>441</height>
              </rect>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QTableWidget" name="sett_categories_unknown_tab">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>460</y>
               <width>1051</width>
               <height>171</height>
              </rect>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="overview_tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Overview</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="overview_graph_hbox">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="month_graph_hbox" stretch="">
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="days_tab">
       <attribute name="title">
        <string>Days</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1071</width>
          <height>691</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTabWidget" name="days_main_tab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::North</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="overall_expenses">
            <attribute name="title">
             <string>Overall expenses</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="day_expenses_vbox"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="balance_tab">
            <attribute name="title">
             <string>Balanace</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="balance_vbox"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="categories_tab">
       <attribute name="title">
        <string>Categories</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1071</width>
          <height>691</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="single_categories_container"/>
       </widget>
      </widget>
      <widget class="QWidget" name="search_tab">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="leftMargin">
           <number>500</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="search_field"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="search_results">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
